packages/millicast-sdk/src/types/index.d.ts

```diff
@@ -714,7 +714,7 @@ declare module '@millicast/sdk' {
      */
     sourceId?: string;
     /**
-     * - Override which events will be delivered by the server ("active" | "inactive").
+     * - Override which events will be delivered by the server ("active" | "inactive" ).
```

Comment:
Is this change correct?

Reply:
Na, I'm sure I added in some text there and removed it lol. Will fix.

Reply:
```suggestion
     * - Override which events will be delivered by the server ("active" | "inactive").
```